en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/C/Continual_Flame.md

```diff
@@ -8,5 +25,5 @@ _2nd-level evocation_
 **Components:** V, S, M (ruby dust worth 50 gp, which the spell consumes) \
 **Duration:** Until dispelled
 
-A magical torch-like flame blazes from one object.
+A magical torch-like flame blazes from one object, providing bright light in a 20-foot radius and dim light for an additional 20 feet.
```

Comment:
Actually neither the WotC version nor the A5e version specify this; however from the WotC version:
> A flame, equivalent in brightness to a torch, springs forth from an object that you touch.

A5e only specifies:
> A magical torch-like flame blazes from the target.

The WotC version means that you should look up the light a torch provides, and then use that. The A5e version is missing that. And I prefer just stating it explicitly, if that's OK by you; that way the spell works by itself without having to check a completely different section of the book.